### PR TITLE
Multiwithdraw

### PR DESCRIPTION
**Pull request by ZmnSCPxj**
Fixes: #2679

### CODE REVIEW

**Comment by ZmnSCPxj** [comment]
Reimplemented as a plugin.

Not sure how this relates to #3867..

File: plugins/spender/multiwithdraw.c [hunk 1]
```diff
@@ -0,0 +1,687 @@
...
+json_multiwithdraw(struct command *cmd,
+		   const char *buf,
+		   const jsmntok_t *params);
+
+const struct plugin_command multiwithdraw_commands[] = {
+	{
+		"multiwithdraw",
+		"bitcoin",
+		"Send to multiple {outputs} via a single Bitcoin transaction.",
+		"Send to multiple {outputs} at optiona {feerate}, spending "
```
Comment (rustyrussell) [inline plugins/spender/multiwithdraw.c:32]: typo: optional

File: plugins/spender/multiwithdraw.c [hunk 2]
```diff
@@ -0,0 +1,687 @@
...
+{
+	struct multiwithdraw_command *mw;
+
+	mw = tal(cmd, struct multiwithdraw_command);
+
+	if (!param(cmd, buf, params,
+		   p_req("outputs", param_outputs_array, &mw->outputs),
+		   p_opt("feerate", param_string, &mw->feerate),
+		   p_opt_def("minconf", param_number, &mw->minconf, 1),
+		   p_opt("utxos", param_string, &mw->utxos),
```
Comment (rustyrussell) [inline plugins/spender/multiwithdraw.c:174]: param_string is ugly here, BTW.  We should have a param_utxos?

**Review by rustyrussell — APPROVED** [review]
Ack c920925216897d3c08c3efb3e638f02bd5c89caf

**Comment by rustyrussell** [comment]
Trivial rebase on top of #3973 (which is about to be merged, once Travis is happy)

**Comment by rustyrussell** [comment]
Ack https://github.com/ElementsProject/lightning/commit/5a6222ca766da7681f3813d7551393657ab93c07

**Pull request merged**